src/conventions.md

```diff
@@ -111,8 +111,8 @@ inspects the Cargo metadata to ensure this.
 
 # How to structure your PR
 
-How you prepare the commits in your PR can make a big difference for the reviewer.
-Here are some tips.
+How you prepare the commits in your PR can make a big difference for the
```

Comment:
Double space here intended? Probably doesn’t matter overmuch.

Reply:
Oh, I just used vim's `gqgq`... I think markdown renders them the same anyway...